src/submission/submission-api.ts

```diff
@@ -138,10 +151,12 @@ class SubmissionController {
       programId: req.params.programId,
     };
     const result = await submission.operations.uploadMultipleClinical(command);
-    if (result.successful) {
-      return res.status(200).send(result);
+    if (!result.successful) {
+      return res.status(422).send({ ...result, fileErrors: errorList });
+    } else if (errorList.length > 0) {
+      return res.status(207).send({ ...result, fileErrors: errorList });
```

Comment:
Literally never seen 207 before. Very interesting.

Reply:
:smile: honestly its just so we have some way to differentiate between the three cases